From a98aa734e0c113d37a971c2b9f09683fd5b77586 Mon Sep 17 00:00:00 2001
From: Ye Li <ye.li@nxp.com>
Date: Sun, 25 Nov 2018 23:10:08 -0800
Subject: [PATCH 5195/5242] MLK-20463 arm64: dts: Disable flexspi in iMX8MM
 EVK M4 DTS

commit  0c50d24e76648f27be17181a0b65f4d1ac02291d from
https://source.codeaurora.org/external/imx/linux-imx.git

M4 will run flexspi XIP image on iMX8MM. Thus, we have to disable flexspi
in M4 dedicated DTS, otherwise the M4 will crash because flexspi probe
in kernel will re-configure the controller.

Signed-off-by: Ye Li <ye.li@nxp.com>
Reviewed-by: Richard Zhu <hongxing.zhu@nxp.com>
(cherry picked from commit 2a370901a663bb518303ed2d5e774f1faa41f5fd)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
index 5d9d239..d6f9085 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
@@ -69,3 +69,7 @@
 &sai1 {
 	status = "disabled";
 };
+
+&flexspi {
+	status = "disabled";
+};
-- 
1.7.9.5

